### ISSUE TITLE
Error in 4.4.0 with rxjs-just rule

### ISSUE THREAD

**Issue by nicolashenry** [cartant/rxjs-tslint-rules#47]
I have this error when I updated to version 4.4.0
```
Error: Cannot find module '@phenomnomnominal/tsquery'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (...\node_modules\rxjs-tslint-rules\dist\rules\rxjsJustRule.js:6:17)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)

```

**Comment by cartant** [comment]
Thanks. I mistakenly added it to `devDependencies` instead of `dependencies`.

**Comment by cartant** [comment]
Should be fixed in 4.4.1. Thanks for opening the issue.
